apps/css-workshop/src/components/Button.astro

```diff
@@ -58,4 +64,13 @@ const {
       </span>
     )
   }
+  {
+    loading && (
+      <div
+        class='iui-progress-indicator-radial iui-button-spinner'
+        data-iui-indeterminate='true'
+        data-iui-size={size === 'small' ? 'x-small' : 'small'}
```

Comment:
If the size of the `ProgressRadial` depends on the size of the Button, should this be done in React as well?

Reply:
fixed in [`a67ebd3`](https://github.com/iTwin/iTwinUI/pull/2058/commits/a67ebd341abb9a17f297b25ea3faa0c3fa85211e)

Reply:
added unit test and tested manually using https://github.com/iTwin/iTwinUI/commit/b2d0a1f0a43de938e0ab1d507330e4afc5db37c7

![image](https://github.com/iTwin/iTwinUI/assets/9084735/720284f1-a46b-4c83-bbcb-224a1cd66c8f)